source/guides/applications/initializers.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Lotus - Guides - Applications Initializers
+---
+
+# Initializers
+
+Each single Lotus application within a project can have one or more custom initializers.
+They are **optional**.
+
+<p class="notice">
+  Initializers are optional
+</p>
+
+They are run **after** the dependencies, the framework and the application code are loaded, but **before** the server or the console are started.
+
+<p class="convention">
+  For a given application named <code>Web</code>, they MUST be placed under <code>apps/web/config/initializers</code>.
+</p>
+
+Their execution order is alphabetical.
```

Comment:
What do you think about suggesting users to follow numbering prefix practice rather relying on alphebet?

That is: `01_my_init.rb` instead of `my_init.rb`?